src/main/webapp/app/exam/manage/students/verify-attendance-check/exam-students-attendance-check.component.html

```diff
@@ -0,0 +1,85 @@
+<div>
+    <div class="d-flex justify-content-between align-items-center border-bottom pb-2 mb-3">
+        <div class="mb-0">
+            <h3>
+                <span jhiTranslate="artemisApp.examManagement.examStudents.verifyChecks">Verify Attendance Check</span>
+                <span class="text-muted">({{ exam.title }})</span>
+            </h3>
+        </div>
+        <div *ngIf="!hasExamStarted">
+            <h3><span class="text-white badge bg-danger" [jhiTranslate]="'artemisApp.examManagement.examStudents.examNotStarted'">Exam have not started yet!</span></h3>
+        </div>
+        <div *ngIf="hasExamStarted">
+            <span [jhiTranslate]="'artemisApp.examManagement.examStudents.notCheckedStudents'">Students who did not sign</span>:
+            <span class="badge bg-primary text-white" id="#registered-users"> {{ allExamUsersAttendanceCheck.length }}</span>
+        </div>
+    </div>
+    <div class="row"></div>
+    <br />
+    <div class="table-responsive" id="exams-table" *ngIf="exam">
```

Comment:
You check for `exam` here but also use `exam.title` above.

Reply:
This page is not accessible if there is not exam so i removed the if check because it does not make sense.